postgres/src/test/scala/zio/sql/postgresql/FunctionDefSpec.scala

```diff
@@ -1066,6 +1067,92 @@ object FunctionDefSpec extends PostgresRunnableSpec with ShopSchema {
       } yield assert(r.head)(equalTo("UTF8"))
 
       assertion.mapErrorCause(cause => Cause.stackless(cause.untraced))
-    } @@ ignore  //todo fix - select(PgClientEncoding())?
+    }
+      @@ ignore, //todo fix - select(PgClientEncoding())?
+    testM("make_date") {
+      val query = select(MakeDate(2013, 7, 15)) from customers
+
+      val expected = LocalDate.of(2013, 7, 15)
+
+      val testResult = execute(query).to[LocalDate, LocalDate](identity)
+
+      val assertion = for {
+        r <- testResult.runCollect
+      } yield assert(r.head)(equalTo(expected))
+
+      assertion.mapErrorCause(cause => Cause.stackless(cause.untraced))
+    },
+    testM("make_interval") {
+      def runTest(interval: Interval) = {
+        val query = select(
+          MakeInterval(
+            interval.years,
+            interval.months,
+            interval.weeks,
+            interval.days,
+            interval.hours,
+            interval.minutes,
+            interval.seconds
+          )
+        ) from customers
+        for {
+          r <- execute(query).to[Interval, Interval](identity).runCollect
+        } yield r.head
+      }
+
+      (for {
+        t1 <- assertM(runTest(Interval()))(equalTo(Interval()))
+        t2 <- assertM(runTest(Interval(days = 10)))(equalTo(Interval(days = 10)))
+        t3 <- assertM(
+                runTest(Interval(years = 10, months = 2, weeks = 1, days = 5, hours = 6, minutes = 20, seconds = 15))
+              )(
+                equalTo(Interval(years = 10, months = 2, weeks = 1, days = 5, hours = 6, minutes = 20, seconds = 15))
+              )
+      } yield t1 && t2 && t3).mapErrorCause(cause => Cause.stackless(cause.untraced))
```

Comment:
this `.mapErrorCause(cause => Cause.stackless(cause.untraced))` is throwing away the tracing information, hence the `[info]     No ZIO Trace available.`. comment this out and it'll show the full stack trace in the test results (which is unideal, and end up doing this - this will be fixed in 2.x I believe)